bundles/org.openhab.binding.blink/src/main/java/org/openhab/binding/blink/internal/dto/BlinkAccount.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.blink.internal.dto;
+
+import static org.openhab.binding.blink.internal.BlinkBindingConstants.*;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * The {@link BlinkAccount} class is the DTO for the login api call.
+ *
+ * @author Matthias Oesterheld - Initial contribution
+ */
+public class BlinkAccount {
```

Comment:
This seems more than a DTO. You could move it to another package and add the null annoation or move the logic into another class.